modules/apps/simplicity-theme/src/templates/portal_normal.ftl

```diff
@@ -0,0 +1,136 @@
+								${site_name}
+							</span>
+						</#if>
+					</h1>
+					<div class="site-description">${site_description}</div>
+					<div class="sign-in-link">
+						<#if !is_signed_in>
+							<a data-redirect="${is_login_redirect_required?string}" href="${sign_in_url}" id="sign-in" rel="nofollow">${sign_in_text}</a>
+						</#if>
+					</div>
+				</div>
+			</div>
+		</div>
+
+		<#if has_navigation && is_setup_complete>
+			<#include "${full_templates_path}/navigation.ftl" />
+		</#if>
+	</header>
+
+	<section id="content">
+		<h1 class="hide-accessible">${the_title}</h1>
+
+		<#if selectable>
+			<@liferay_util["include"] page=content_include />
+		<#else>
+			${portletDisplay.recycle()}
+
+			${portletDisplay.setTitle(the_title)}
+
+			<@liferay_theme["wrap-portlet"] page="portlet.ftl">
+				<@liferay_util["include"] page=content_include />
+			</@>
+		</#if>
+	</section>
+
+	<footer id="footer" role="contentinfo">
+		<div class="footer-wrap">
+			<aside class="footer-area">
+				<div class="footer-area-column-1">
+					<section class="footer-text">
+						<h2 class="footer-title">
+							<@liferay.language key="footer-contact-title" />
+						</h2>
+						<div class="footer-text-area">
+							<p>
+								<@liferay.language key="footer-contact-description" />
+							</p>
+						</div>
+					</section>
+				</div>
+				<div class="footer-area-column-2">
+					<section class="footer-text">
+						<h2 class="footer-title">
+							<@liferay.language key="footer-about-title" />
+						</h2>
+						<div class="footer-text-area">
+							<p>
+								<@liferay.language key="footer-about-description" />
+							</p>
+						</div>
+					</section>
+				</div>
+			</aside>
+			<nav class="social-media-navigation">
+				<ul class="social-media-links">
+					<li id="menu-item-facebook" class="menu-item menu-item-facebook">
+						<a href="http://www.facebook.com" class="social-facebook">
+							<svg aria-hidden="true" class="lexicon-icon lexicon-icon-add-column lexicon-icon-facebook" >
+								<use xlink:href="${themeDisplay.getPathThemeImages()}/clay/icons.svg#social-facebook" />
+							</svg>
+						</a>
+					</li>
+					<li id="menu-item-twiiter" class="menu-item menu-item-twiiter">
+						<a href="http://www.twitter.com" class="social-twitter">
+							<svg aria-hidden="true" class="lexicon-icon lexicon-icon-add-column lexicon-icon-twitter">
+								<use xlink:href="${themeDisplay.getPathThemeImages()}/clay/icons.svg#twitter" />
+							</svg>
+						</a>
+					</li>
+					<li id="menu-item-linkedin" class="menu-item menu-item-linkedin">
+						<a href="http://www.linkedin.com" class="social-linkedin">
+							<svg aria-hidden="true" class="lexicon-icon lexicon-icon-add-column lexicon-icon-linkedin">
+								<use xlink:href="${themeDisplay.getPathThemeImages()}/clay/icons.svg#social-linkedin" />
+							</svg>
+						</a>
+					</li>
+				</ul>
```

Comment:
We should make these social links a theme setting to allow the end user to be able to turn them on and off rather than hardcoding all of them to show up on the site all the time.